data/models/common_base_ingredient.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+
+
+class CommonBaseIngredient(models.Model):
+    class Meta:
+        abstract = True
+
+    creation_date = models.DateTimeField(auto_now_add=True)
+    modification_date = models.DateTimeField(auto_now=True)
+    name = models.TextField()
+    name_en = models.TextField(blank=True)
+    public_comments = models.TextField(null=True, blank=True, verbose_name="commentaires publics")
```

Comment:
Pour suivre le même standard que les autres, ça t'irait d'avoir seulement le `blank=True` ?
```suggestion
    public_comments = models.TextField(blank=True, verbose_name="commentaires publics")
```